src/Components/RandomQuestionList/QuestionItemS.scss

```diff
@@ -0,0 +1,19 @@
+.QuestionItem {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-wrap: wrap;
+    margin-bottom: 1rem;
+    width: 50rem;
+    border: 2px solid black;
+    background-color: lightgray;
+    padding: 1rem;
```

Comment:
i know the hard-coded styling is no good, will fix once i learn the proper way to style